pyproject.toml.in

```diff
@@ -0,0 +1,17 @@
+[build-system]
+requires = ["sip >=6, <7"]
+build-backend = "sipbuild.api"
+
+[tool.sip.metadata]
+name = "pySavitar"
+
+[tool.sip.project]
+builder-factory = "CMakeBuilder"
+sip-files-dir = "${CMAKE_CURRENT_SOURCE_DIR}/python/"
+sip-include-dirs = ["CMAKE_CURRENT_SOURCE_DIR/python/"]
+build-dir = "${CMAKE_CURRENT_BINARY_DIR}/pySavitar/"
+
+[tool.sip.bindings.pySavitar]
+exceptions = true
+release-gil = true
+concatenate = 8
```

Comment:
Why 8? Is there a reason to not leave this at whatever the default is?

Reply:
Ah, wait. Looking back it makes sense. THe CMAKE does mention something about touching the 8 files. Perhaps it's a good idea to add comments here and in the cmake to say that these two things are talking about the same 8 files? I can imagine that this might cause issues in the future.

Reply:
Yes will update that.

Reply:
Because those numbers in the SIPMacros and the pyproject.toml.in should be the same. in all cases.